.github/workflows/release.yaml

```diff
@@ -43,6 +43,43 @@ jobs:
       secret_registry: ${{ matrix.secret_registry }}
     secrets: inherit
 
+  multiarch:
```

Comment:
Should we be testing this step as well in the nightly release validation?

Reply:
https://github.com/rancher-sandbox/rancher-turtles/pull/235 is now aligned with these changes.